cogs/sessions.py

```diff
@@ -0,0 +1,72 @@
+from asyncio import tasks
+
+from dateutil import parser as timeinput
+from dateutil.parser import ParserError
+from discord import app_commands
+from discord.app_commands import commands
+
+from components.bot import RecruitBot
+from components.config.config_manager import configInstance
+from components.recruitment import recruit
+
+
+# Object representing a recruitment session (user has requested reminders on an interval)
+class Session:
+    ctx: commands.Context
+    interval: int = 30
+    active: bool
+    bot: RecruitBot
+
+    @tasks.loop(seconds=interval)
```

Comment:
this will kind of work, but the decorator can't take runtime defined arguments, so the interval will only ever be 30 in this case.